tests/Library/Vanilla/EmbeddedContent/Embeds/FileEmbedTest.php

```diff
@@ -45,7 +46,7 @@ public function testLegacyDataFormat() {
 JSON;
 
         $oldData = json_decode($oldDataJSON, true);
-        $dataEmbed = new FileEmbedTest($oldData);
-        $this->assertInstanceOf(FileEmbedTest::class, $dataEmbed);
+        $dataEmbed = new FileEmbed($oldData);
```

Comment:
@charrondev can you confirm that this is a correct change? The other way looks like a typo.

Reply:
Yup, you’ve got it the right way.